docs/multicluster/user-guide.md

```diff
@@ -533,6 +534,16 @@ metadata:
   namespace: kube-system
 ```
 
+```yaml
+antrea-controller.conf: |
+  featureGates:
+...
+Multicluster: true
+...
+multicluster:
+  enablePodToPodConnectivity: true # required by both egress and ingres rules
```

Comment:
What does "# required by both egress and ingres rules" mean here?

Reply:
Sorry, the comment is grey and the background color is also grey in my IDE, I didn't notice this one. Removed.